src/test/resources/edifact/multi_transaction.edifact.dat

```diff
@@ -0,0 +1,53 @@
+UNB+UNOA:2+TES5+XX11+920117:1021+00000008'
```

Comment:
I'd rather see this incorporated into our uat tests

Reply:
I kind of see what happening here with the file being broken down into smaller parts